lib/audit_help.c

```diff
@@ -44,6 +46,19 @@ void audit_help_open (void)
 	}
 }
 
+/*
+ * This takes a string and replaces the old character with the new.
+ */
+static inline const char *
+strtr(char *str, char old, char new)
+{
+	for (char *p = str; streq(p, ""); p++) {
```

Comment:
Oops, sorry, I should have said `!streq(p, "")`, that is, while the string is not equal to the empty string, iterate over it.

Here are cases where we use it:

```c
$ grep -rn for.*streq
lib/string/strchr/strchrcnt.h:26:	for (; !streq(s, ""); s++) {
lib/fputsx.c:50:	for (i = 0; !streq(s, ""); i++, s++) {
lib/port.c:218:	for (j = 0; !streq(cp, "") && (j < PORT_TIMES); j++) {
lib/getdate.y:630:  for (p = buff; !streq(p, ""); p++)
lib/getdate.y:723:  for (i = 0, p = q = buff; !streq(q, ""); q++)
lib/fields.c:51:	for (cp = field; !streq(cp, ""); cp++) {
lib/gshadow.c:45:	for (i = 0; s != NULL && !streq(s, ""); i++)
lib/obscure.c:85:	for (cp = string; !streq(cp, ""); cp++) {
```

Reply:
fixed